site/js/modules/webui/input.js

```diff
@@ -340,6 +340,7 @@ class FieldInstance {
 
     // Get the value from the result.
     // Throws an error on validation errors.
+    // All result values without content should return undefined.
```

Comment:
This is a bit too board (and I think incorrect).
Does an unchecked checkbox have content?

This description will probably need to be expanded.